contrib/pg_tde/src/catalog/tde_keyring_parse_opts.c

```diff
@@ -274,11 +274,33 @@ json_kring_object_end(void *state)
 			/* We're done parsing the top level object */
 			break;
 		case JK_EXPECT_EXTERN_VAL:
+			if (!parse->field_type)
+				ereport(ERROR,
```

Comment:
See my earlier comment about `errcode(ERRCODE_INVALID_PARAMETER_VALUE)` for all of these.